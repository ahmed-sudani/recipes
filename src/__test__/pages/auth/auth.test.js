import { render } from '@testing-library/react'
import SignIn from '../../../pages/auth/signin'
import VerifyRequest from '../../../pages/auth/verifyRequest'

describe('Auth', () => {
  it('signIn should render correctly', () => {
    const { container } = render(<SignIn />)
    expect(container).toMatchSnapshot()
  })
  it('verifyRequest should render correctly', () => {
    const { container } = render(<VerifyRequest />)
    expect(container).toMatchSnapshot()
  })
})
