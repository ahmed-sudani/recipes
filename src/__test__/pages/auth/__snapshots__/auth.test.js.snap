// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth signIn should render correctly 1`] = `
<div>
  <form
    action="/api/auth/signin/email"
    class="Form"
    method="POST"
  >
    <div
      class="Text"
    >
      Login To Start Add and Save Your Favorites Recipes
    </div>
    <div
      class="inputForm"
    >
      <input
        name="csrfToken"
        type="hidden"
        value=""
      />
      <label>
         Email 
      </label>
      <input
        class="input"
        id="email"
        name="email"
        required=""
        type="email"
      />
    </div>
    <button
      class="btn"
      name="Send Link"
    >
      Send Link
    </button>
  </form>
</div>
`;

exports[`Auth verifyRequest should render correctly 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      Check your email
    </h1>
    <p>
       A sign in link has been sent to your email address.
    </p>
    <span />
  </div>
</div>
`;
